resources/public/contracts/package.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
1. Why is this not specified as lein npm dependency in project file?
2. I'd expect instructions in README on how to make style check and style fix (and I'd probably make eponymous single commands to do this for both clojure and solidity files)

Reply:
1) I wanted it to be close to solidity files, but on the second thought having just one package json is better.
2) Good point. Although, there is nothing complicated. I asked our grandmasters a question regarding the linter, but I guess it's not a standard to have a linter tool for clojure files :/

Reply:
1. I would put this a bit differently: we should have just one point for managing all dependencies and that is lein + project.clj. The various plugins such as lein npm exist exactly for the reason. The lein npm plugins creates a package.json temporarily, but that's internal workings we shouldn't be interested in as consumers.
2. Even if you just clicked something in IntelliJ to format Clojure, I'd like it documented so that e.g. I can to it too